### DISCUSSION TITLE
User Survey completion page needs updating

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2185] [Documentation]
```
User Survey
Thanks - your feedback is much appreciated!

And if you want to help out (coding, design, translation, testing...), or just share video or audio of your best sessions, let us know on the Jamulus SourceForge forums!
```
Should probably point to the project's github discussions link instead.

**Comment by ann0see** [comment]
@gilgongo should be able to update it. Probably we need to update the survey anyway. @mulyaj was somewhere involved

**Comment by gilgongo** [comment]
Thanks - now updated. And yes, I was trying to find a better survey system and revise the questions.
